components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIManager.java

```diff
@@ -278,6 +278,15 @@ void addOrUpdateComplexityDetails(String uuid, GraphqlComplexityInfo graphqlComp
      */
     void addSubscriber(String username, String groupingId) throws APIManagementException;
 
+    /**
+     * Creates a new subscriber given the username, the grouping Id and organization
+     *
+     * @param username   Username of the subscriber to be added
+     * @param groupingId - the groupId to which the subscriber belongs to
+     * @throws org.wso2.carbon.apimgt.api.APIManagementException if failed add subscriber
```

Comment:
Shall we add organization parameter description? Also let's add a "." at the end of method description and do the necessary changes to keep parameter descriptions consistent.